src/main/java/seedu/address/model/Model.java

```diff
@@ -129,4 +130,13 @@
      */
     Person getPerson(Index targetIndex);
 
+    /**
+     * Sets the selected event.
+     */
```

Comment:
would be good to add @param tag in the javadoc comment.

Reply:
Actually not sure if this is strictly required? If so we have to mass add params for every method...